mermaid-hs.cabal

```diff
@@ -85,6 +85,7 @@ library
   hs-source-dirs:     src
   exposed-modules:
     FlowChart
+    Parser
```

Comment:
Something like this:

```
src/Diagram.hs
src/Diagram/FlowChart.hs
src/Diagram/GanttChart.hs
```

Reply:
do you mean that I shan't expose `Parser`?